src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputList) {
+        inputList.remove(0);

```

Comment:
Directly modifying the input parameter (inputList) by removing the first element can lead to unexpected behavior if the same list is used elsewhere. It would be safer to skip the first line during the stream processing.